configs/vitest.config.ts

```diff
@@ -22,6 +22,7 @@ export default defineConfig(({ mode }) => {
       isolate: !testSSR,
       passWithNoTests: true,
       environment: testSSR ? "node" : "jsdom",
+      deps: { registerNodeLoader: false },
```

Comment:
What's that for?

Reply:
Unfortunately, the node loader is broken in node@20 - and it is still enabled by default in the solid plugin, because it was supposed to fix the problems it now causes. Fortunately, they are solved in vite's internal loader now so that it's fine to disable it on older node versions as well.